maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -557,6 +564,20 @@ class Orchestra(
         return false
     }
 
+    private fun startRecordingCommand(command: StartRecordingCommand): Boolean {
+        val pathStr = command.path + ".mp4"
```

Comment:
Lets use the exact path as specified, not appending the extension.
Otherwise `screenRecording: login.mp4` would be recording `login.mp4.mp4`.

Reply:
I think changing the behaviour of screenshot over this does not bring enough value to be worth it.
Lets keep the ".mp4" addition extension as in the PR now to be in line with screenshot.